python/social_bot/envs/play_ground.py

```diff
@@ -294,8 +281,6 @@ def _get_camera_observation(self):
             image = PIL.Image.fromarray(image).resize(self._resized_image_size,
                                                       PIL.Image.ANTIALIAS)
             image = np.array(image, copy=False)
-        if self._data_format == "channels_first":
```

Comment:
channel problem of tf2.0 is fixed, so remove this option

Reply:
Does this mean PlayGround will always output in the format "channels_last"?